testcontainers-dapr/src/main/java/io/dapr/testcontainers/MetadataEntry.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2024 The Dapr Authors
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package io.dapr.testcontainers;
+
+public class MetadataEntry {
+  private String name;
+  private Object value;
+
+  public MetadataEntry(String name, Object value) {
+    this.name = name;
+    this.value = value;
+  }
+
+  public String getName() {
+    return name;
+  }
+
+  public Object getValue() {
+    return value;
+  }
+
+  public void setName(String name) {
+    this.name = name;
+  }
+
+  public void setValue(Object value) {
+    this.value = value;
+  }
```

Comment:
Optionally, add extra methods to return the value as a specific type, for example: getValueAsBoolean()

Reply:
we can add this when we have a scenario to cover that.